eth-custodian/test/EthCustodianProxy.js

```diff
@@ -0,0 +1,266 @@
+            )
+                .to.emit(ethCustodian, 'Deposited')
+                .withArgs(ethCustodianProxy.address, protocolMessage, amountToTransfer, 0);
+
+            const balanceAfter = ethers.BigNumber.from(
+                await ethers.provider.getBalance(ethCustodian.address));
+            const balanceDiff = balanceAfter.sub(balanceBefore);
+            expect(balanceDiff).to.equal(amountToTransfer);
+        });
+
+        it('to Near Should change the balance and emit the event', async () => {
+            const nearRecipientAccountId = 'recipient.near';
+
+            const amountToTransfer = 50000;
+            const balanceBefore = ethers.BigNumber.from(
+                await ethers.provider.getBalance(ethCustodian.address));
+
+            const options = { value: amountToTransfer };
+            await expect(
+                ethCustodianProxy
+                    .connect(user1)
+                    .depositToNear(nearRecipientAccountId, options)
+            )
+                .to.emit(ethCustodian, 'Deposited')
+                .withArgs(ethCustodianProxy.address, nearRecipientAccountId, amountToTransfer, 0);
+
+            const balanceAfter = ethers.BigNumber.from(
+                await ethers.provider.getBalance(ethCustodian.address));
+            const balanceDiff = balanceAfter.sub(balanceBefore);
+            expect(balanceDiff).to.equal(amountToTransfer);
+        });
+    });
+
+    describe('Pause', () => {
+        it('Should pause deposit to NEAR', async () => {
+            await ethCustodianProxy.pauseProxy(PAUSED_DEPOSIT_TO_NEAR);
+
+            await expect(ethCustodianProxy.depositToNear('recipient.near', { value: 50000 }))
+                .to.be.revertedWith('Pausable: paused');
+        });
+
+        it('Should pause deposit to EVM', async () => {
+            await ethCustodianProxy.pauseProxy(PAUSED_DEPOSIT_TO_EVM);
+
+            await expect(ethCustodianProxy.depositToEVM(ethRecipientOnNear.address, { value: 50000 }))
+                .to.be.revertedWith('Pausable: paused');
+        });
+
+        it('Should pause withdraw', async () => {
+            await ethCustodianProxy.migrateToNewProofProducer(newProofProducerData, migrationBlock);
+            await ethCustodianProxy.pauseProxy(PAUSED_WITHDRAW);
+            const proof = require('./proof_template_from_testnet.json');
+
+            await expect(ethCustodianProxy.withdraw(borshifyOutcomeProof(proof), migrationBlock + 1))
+                .to.be.revertedWith('Pausable: paused');
+        });
+
+        it('Should pause withdraw pre-migration', async () => {
+            await ethCustodianProxy.migrateToNewProofProducer(newProofProducerData, migrationBlock);
+            await ethCustodianProxy.pauseProxy(PAUSED_WITHDRAW_PRE_MIGRATION);
+            const proof = require('./proof_template_from_testnet.json');
+
+            await expect(ethCustodianProxy.withdraw(borshifyOutcomeProof(proof), 1099))
```

Comment:
use const for 1099

Reply:
Done c457ab171ed19820c6a79e80d8dc85e00f62a65d